### DISCUSSION TITLE
FAILURE REASON TIMEOUT

### DISCUSSION THREAD

**Discussion by HotCrypt** [lightningnetwork/lnd#8324] [Q&A]
Hi, I'm still experimenting with LND to fully understand how the network works and keep my lnd node 24/7 well connected and well configured, so I'm trying to balance a channel (with 1ML) manually but I get an error after the fly attempts (for info I have the same error too with rebalance.py):

`lncli addinvoice 119000`

`lncli payinvoice --allow_self_payment --last_hop 0217890e3aad8d35bc054f43acc00084b25229ecff0ab68debd82883ad65ee8266 --outgoing_chan_id 897377410125987841 --pay_req lnbc1190u1pjclks3pp5j9hh07l4hz6zhxkhnfkauzjrfl98ukpe0ss8ts4mdqzzlmtffy5qdqqcqzzsxqyz5vqsp5fm425ejzw8ty837uzpv9kep23jn3drjxwjznnlpzpdzelv7z2hwq9qyyssql4e3sqz4wn5t40l5r65ngfwed86aq8yd6wtjlz9nqrqd9m3y0929eemw2vy27mjekvxks3qsjneu3ggmeyf2xzhfgcvfe6268sn4xvgq9dypgs
`
> Payment hash: 916f77fbf5b8b42b9ad79a6dde0a434fca7e58397c2075c2bb68042fed694928
> Description: 
> Amount (in satoshis): 119000
> Fee limit (in satoshis): 5950
> Destination: 0368083d4e4705f261e365cb29bb65299e7c62fe3953b3f4820efce5af89b737be
> Confirm payment (yes/no): yes

```
| TEMPORARY_CHANNEL_FAILURE @ 1st hop |        3.357 |        5.751 | 119000       | 1.119 |   823658 | 897377410125987841 | 1ML.com node ALPHA->BitShirt |
+-------------------------------------+--------------+--------------+--------------+-------+----------+--------------------+------------------------------+
Amount + fee:   0 + 0 sat
Payment hash:   916f77fbf5b8b42b9ad79a6dde0a434fca7e58397c2075c2bb68042fed694928
Payment status: FAILED, reason: FAILURE_REASON_TIMEOUT
[lncli] FAILED
```

My command is ok? I tried to increase the fees but same error.
Could you help me to succeed please?

**Comment by microlancer** [comment]
Take a look at the channels between you and your peers. You can use a network browser like this one https://amboss.space/node/0368083d4e4705f261e365cb29bb65299e7c62fe3953b3f4820efce5af89b737be?section=Channels

And then, take a look at the different sizes of channels, max_htlc_msat, fee ppm, and other bits of info. If there are only small channels, then it's going to be tougher to route. But also, sometimes big channels are hard to route as well if they are drained in one direction (like all the liquidity on one side). If you're trying to go in the opposite direction it's fine, but in the same direction, there's no liquidity left so you'll get "temporary failure" on the route. You can try again later, to see if the liquidity changes, so it's temporary.

The timeout happens because it kept trying to find routes but it takes time often in the situation where there's not enough liquidity. You can increase the timeout `--timeout 5m` to make it longer, but usually that doesn't help for me personally, because usually if it's drained, it's drained.

So it could just be the case in your situation where there's just not enough available liquidity on the network in the direction you're trying to rebalance. People might suggest you open more channels or buy more incoming liquidity from others.

Sometimes increasing fees help, but if there's literally not enough liquidity, it doesn't help because people don't have the sats in the channels that you want to send through them.

**Comment by HotCrypt** [comment]
Thx, very good explanation. Yes I was trying to increase incoming liquidity(remote) with 1ML, I think my node is not well connected, I will open larger channels but it is not the right time with the fees so high on the On-Chain network (200sat/vb), 30sat/vb avg is better. 
[ I don't know what's going on with the Btc network but I no longer do any transactions because it's too expensive, I hope the devs will find a solution to remedy this, reminds me a lot of eth and its exorbitant fees... ]
```
mainnet                            Commit ------- Balance --------- --- Fee ----
Alias or Pubkey           Capacity   Fee        Local       Remote   Base PerMil
-------------------- ------------- ------ ------------ ------------ ----- ------
1ML.com node ALPHA          271745  49476       221939            0     0      0
```

```
Channel ID:       897377410125987841
Alias:            1ML.com node ALPHA
Pubkey:           0217890e3aad8d35bc054f43acc00084b25229ecff0ab68debd82883ad65ee8266
Channel Point:    5abbb8dfbecffdc45a845bd8517cce34bd40f2e8d585144f9c7778b4b7973891:1
Local ratio:      1.000
Fee rates:        0ppm (own), 1ppm (peer)
Capacity:            271,745
Remote available:          0
Local available:     219,222
Rebalance amount:   -109,611


```
